_maps/configs/pgf_crying_sun.json

```diff
@@ -21,29 +21,30 @@
 			"officer": true,
 			"slots": 1
 		},
-		"Bridge Crew": {
-			"outfit": "/datum/outfit/job/gezena/assistant/bridge",
+		"Marine Lieutenant": {
+			"outfit": "/datum/outfit/job/gezena/hos",
+			"officer": true,
 			"slots": 1
 		},
-		"Navy Engineer": {
-			"outfit": "/datum/outfit/job/gezena/engineer",
-			"slots": 2
-		},
 		"Navy Doctor": {
 			"outfit": "/datum/outfit/job/gezena/doctor",
 			"slots": 1
 		},
-		"Crewman": {
-			"outfit": "/datum/outfit/job/gezena/assistant",
+		"Navy Engineer": {
+			"outfit": "/datum/outfit/job/gezena/engineer",
 			"slots": 2
 		},
-		"Marine Sergeant": {
-			"outfit": "/datum/outfit/job/gezena/hos",
+		"Helmsman": {
+			"outfit": "/datum/outfit/job/gezena/hop",
```

Comment:
change this back to assistant/bridge or change that to another version. I'd also like to see what all you put in their locker, because I don't really think they need one.